controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -147,8 +171,11 @@ def main() -> None:
 
     remove_unused_robots(supervisor)
 
-    with record_animation(supervisor, REPO_ROOT / 'recordings' / recording_path()):
-        run_match(supervisor)
+    recording_path = REPO_ROOT / 'recordings' / get_recording_path()
```

Comment:
Should `get_recording_path` just return the fully-formed path? If not I wonder if there's a better name for it?

Reply:
Maybe? Do you have a suggestion for a better name?

I've also been thinking about whether this sort of thing should be more configurable. I think eventually I'd like to get to a point where there's either a configuration file and/or suite of environment variables which we can use to control where things like the recordings end up in order to more completely automated the running of the competition. I'm therefore not especially bothered about these being imperfect for now (i.e: "good enough" is fine).